packages/ra-ui-materialui/src/input/ArrayInput/ArrayInput.tsx

```diff
@@ -71,9 +71,7 @@ import { ArrayInputContext } from './ArrayInputContext';
 export const ArrayInput = (props: ArrayInputProps) => {
     const {
         className,
-        defaultValue,
         label,
-        isFetching,
```

Comment:
Sorry I should have commented. The prop wasn't used. However just realised that by destructuring it will mean its inadvertingly passed into any component that uses the "rest" of the props. Will revert.